File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int check=0; //짝수면 0
        if(n%2==1){
            check=1; //홀수면 변경으로 1
        }
        for(int i=0; i<=n; i++){
            if(check==1){ //홀수
                if(i%2==1){
                    answer+=i;
                }
            }
            else if (i%2==0){
                answer+=i*i;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.py
def solution(n):
    answer = 0
    check = 0
    if n%2==1 :
        check=1
    for i in range(0,n+1) :
        if check==1 :
            if i%2==1 :
                answer+=i
        elif i%2==0 :
            answer+=i*i;
    return answer